src/kirby/components/modal/services/modal-closer-service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ModalCloserService {
```

Comment:
I guess this one is needed because of cyclic references or...?

Reply:
exactly

Reply:
Please investigate if you can retrieve the `ModalService` by injecting the `Injector` and then getting ti from there, eg.: `injector.get(ModalService)`